src/Components/ArtworkGrid/ArtworkGrid.tsx

```diff
@@ -60,6 +65,20 @@ export class ArtworkGridContainer extends React.Component<
     }
   }
 
+  columnBreakpoints = memoize(createColumnBreakpoints, isEqual)
```

Comment:
The intent might be slightly clearer in the future if `memoize` was renamed to `memoizeOnce`